dev_guide/pod_preset.adoc

```diff
@@ -0,0 +1,565 @@
+[[dev-guide-pod-presets]]
+= Injecting Information into Pods Using Pod Presets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _pod preset_ is an object that injects user-specified information into pods as they are created.
+
+//tag::pod-preset[]
+[IMPORTANT]
+====
+Pod presets is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+Using pod preset objects you can inject:
+
+* xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret objects]
+* xref:../dev_guide/configmaps.adoc#consuming-configmap-in-pods[`ConfigMap` objects]
+* xref:../dev_guide/volumes.adoc#dev-guide-volumes[storage volumes]
+* container volume mounts
+* environment variables
+
+Developers only need to add the appropriate label to the pod specification in order to add all that information to the pod. The xref:../architecture/core_concepts/pods_and_services.adoc#labels[label] on a pod associates the pod with one or more pod preset objects that have a matching xref:../architecture/core_concepts/pods_and_services.adoc#services[label selectors].
```

Comment:
I would phrase this as, developers need to ensure the pod's labels match the label selector on the PodPreset.